packages/server-admin-ui/src/views/appstore/appStore.scss

```
@@ -1,45 +1,60 @@
-      }
-      i {
+  .message__container {
+    padding: 10px;
+    color: var(--bs-warning-text-emphasis, #9d8641);
+    border: 1px solid var(--bs-warning-border-subtle, #9d8641);
+    background-color: var(--bs-warning-bg-subtle, #fff2cd);
+    p {
+      span {
         margin-right: 5px;
       }
+      margin: 0;
```

Stylelint (17.11.1)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)